cmake/radium_setup_functions.cmake

```diff
@@ -500,8 +594,52 @@ function(configure_radium_plugin)
     endif ()
 endfunction()
 
+# ---------------------------------------------------------------------------------------------
 
-include(CMakePackageConfigHelpers)
+# mandatory parameters :
```

Comment:
All other functions have a brief description, but this one is missing. please add it.